config/items.conf.php

```diff
@@ -77,6 +77,9 @@
                     'type' => 'keyword',
                     'ignore_above' => 256,
                 ],
+                'referenced_resources' => [
```

Comment:
shall we call it `asset_uris`? Cause an item can reference to a test, media, workflow, etc. But our use case is now only interested in assets. It might bring confusion in the future and I do not see any use case where is useful to aggregate all the related resources in a single array. Wdyt?

Reply:
```suggestion
                'asset_uris' => [
```